.travis.yml

```diff
@@ -34,53 +36,36 @@ matrix:
 
     ## ** Builds that are published **
 
-    # linux cfi build node v6/release
+    # linux cfi build node v10/release
```

Comment:
As we discussed earlier, you can remove these v10 and other references to exact node version. It should be enough that it is defined in the start of this .travis.yml.

Reply:
yeah this was the cfi build that I wasn't sure who's using but I guess it doesn't hurt to use node 12 for this as well since they are not automatically pulled

Reply:
d8e9428210c638467a5a289d6c5f6cc3f6c38e28